ANSWERS.md

```diff
@@ -1,4 +1,16 @@
 - [ ] Why would you use class component over function components (removing hooks from the question)?
+
```

Comment:
If we're assuming that hooks don't exist there are two main reasons you'd use class components over functional components. One is lifecycle methods like you mentioned. The other is that functional components couldn't hold state before hooks. You had to use a class if you wanted to use state. 👍